### PR TITLE
Creates testing script for exporting a Habitat...

### PR DESCRIPTION
**Pull request by nellshamrell**
Package to a tarball.

Adds end to end test for exporting a tar file (ty for the model, @baumanj!)

In preparation for #4172

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: test/end-to-end/test_tar_export [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    print_help
+    echo
+    echo "<PKG_IDENT> must be specified"
+    exit 1
+else
+    pkg_ident="$1"
+fi
+
+# Remove tarball if already present
+[ -e *.tar.gz ] && rm *.tar.gz
```
Comment (baumanj) [inline test/end-to-end/test_tar_export]: This can be written as `rm -f *.tar.gz`, which I find a little simpler.

**Comment by baumanj** [comment]
Nice! The more of these kinds of tests we get, the more value there'll be in getting them into our automated test suites.

**Comment by baumanj** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/318116748).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/318116748).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**